mlflow/langchain/__init__.py

```diff
@@ -270,6 +273,19 @@ def load_retriever(persist_directory):
         _MODEL_TYPE_KEY: lc_model.__class__.__name__,
         **model_data_kwargs,
     }
+
+    if isinstance(lc_model, langchain.chains.RetrievalQA):
```

Comment:
this needs to work for an arbitrary LCEL chain too. Ideally, it would be great if we could inspect any chain, not just RetrievalQA

Reply:
Supported!